staging/src/k8s.io/client-go/restmapper/shortcut_test.go

```diff
@@ -255,6 +259,148 @@ func TestKindForWithNewCRDs(t *testing.T) {
 	}
 }
 
+func TestWarnAmbigious(t *testing.T) {
```

Comment:
can you add a testcase that includes multiple ambiguous groups with multiple versions, that captures all the warnings that would be output in that case, to exercise the deduplication by version and group?

Reply:
@liggitt I've added test cases for multiple-resource with multiple-versions. Could you PTAL?.